--- libsidplayfp-1.8.8/builders/hardsid-builder/hardsid-emu-unix.cpp.orig	2018-08-13 16:04:41.571917491 +0100
+++ libsidplayfp-1.8.8/builders/hardsid-builder/hardsid-emu-unix.cpp	2018-08-13 16:04:47.591931489 +0100
@@ -215,7 +215,7 @@
     else
     {
         m_accessClk += cycles;
-        ioctl(m_handle, HSID_IOCTL_DELAY, static_cast<uint>(cycles));
+        ioctl(m_handle, HSID_IOCTL_DELAY, static_cast<unsigned int>(cycles));
         m_context->schedule (*this, HARDSID_DELAY_CYCLES, EVENT_CLOCK_PHI1);
     }
 }
